tests/functional/test_new_resolver.py

```diff
@@ -1,10 +1,13 @@
 import json
 
+import os
+
 import pytest
 
 from tests.lib import (
     create_basic_sdist_for_package,
     create_basic_wheel_for_package,
+    create_test_package_with_setup,
```

Comment:
Hmm, I wonder whether I could have used this for my previous PR, rather than implementing `create_basic_sdist_for_package`. I should take a look at that.

Reply:
Oh boi, we really need to clean up our test helpers.

Reply:
OH WAIT, it's a GSoC Project!

https://github.com/pypa/packaging.python.org/wiki/GSoC-2020-Ideas-Page#project-ideas

(yes, it was all a setup) :P